ReactWindows/ReactNative.Shared/Chakra/Executor/ChakraJavaScriptExecutor.cs

```diff
@@ -283,7 +283,7 @@ private JToken ConvertJson(JavaScriptValue value)
 #else
         private JavaScriptValue ConvertJson(JToken token)
         {
-            var jsonString = token.ToString(Formatting.None);
+            var jsonString = token?.ToString(Formatting.None) ?? null;
```

Comment:
Can't you remove `?? null`?

Reply:
I was thinking of Matt's suggestion and used null by mistake...

---
In reply to: [192273111](https://github.com/Microsoft/react-native-windows/pull/1847#discussion_r192273111) [](ancestors = 192273111)